src/mono/wasm/runtime/CMakeLists.txt

```
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.14.5)
+
+project(mono-wasm-runtime)
```

a nit, in `src/libraries/Native/**/CMakeLists.txt` we specify the (only) language in use:
```suggestion
project(mono-wasm-runtime C)
```
so cmake does not waste time looking for the unused C++ compiler and its configs.